app/server/api/transfer_usage_api.py

```diff
@@ -19,19 +35,23 @@ def post(self):
         icon = post_data.get('icon')
         priority = post_data.get('priority')
         # EG {'fr': 'fromage'}
-        translations = post_data.get('translations')
 
         if name is None or icon is None:
             return make_response(jsonify({'message': 'Name and Icon must not be None'})), 400
 
         try:
-            usage = TransferUsage(name=name, translations=translations, icon=icon, priority=priority)
```

Comment:
What happened to translations? I don't think the general python i18n module handles them?

Reply:
Can they be removed?

Reply:
nah we still need them for I18n on the android app